charts/vector/values-agent.yaml

```diff
@@ -0,0 +1,651 @@
+# Default values for Vector
+# See Vector helm documentation to learn more:
+# https://vector.dev/docs/setup/installation/package-managers/helm/
+namespaceOverride: ""
+# nameOverride -- Override the name of resources.
+nameOverride: ""
+
+# fullnameOverride -- Override the full name of resources.
+fullnameOverride: ""
+
+# role -- [Role](https://vector.dev/docs/setup/deployment/roles/) for this Vector instance, valid options are:
+# "Agent", "Aggregator", and "Stateless-Aggregator".
+
+# Each role is created with the following workloads:
+# Agent = DaemonSet
+# Aggregator = StatefulSet
+# Stateless-Aggregator = Deployment
+role: "Agent"
+
+# rollWorkload -- Add a checksum of the generated ConfigMap to workload annotations.
+rollWorkload: true
+
+# commonLabels -- Add additional labels to all created resources.
+commonLabels: {}
+
+# Define the Vector image to use.
+image:
+  # image.repository -- Override default registry and name for Vector's image.
+  repository: timberio/vector
+  # image.pullPolicy -- The [pullPolicy](https://kubernetes.io/docs/concepts/containers/images/#image-pull-policy) for
+  # Vector's image.
+  pullPolicy: IfNotPresent
+  # image.pullSecrets -- The [imagePullSecrets](https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod)
+  # to reference for the Vector Pods.
+  pullSecrets: []
+  # image.tag -- The tag to use for Vector's image.
+  # @default -- Derived from the Chart's appVersion.
+  tag: ""
+  # image.sha -- The SHA to use for Vector's image.
+  sha: ""
+
+# replicas -- Specify the number of Pods to create. Valid for the "Aggregator" and "Stateless-Aggregator" roles.
+replicas: 1
+
+# podManagementPolicy -- Specify the [podManagementPolicy](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#pod-management-policies)
+# for the StatefulSet. Valid for the "Aggregator" role.
+podManagementPolicy: OrderedReady
+
+# Create a Secret resource for Vector to use.
+secrets:
+  # secrets.generic -- Each Key/Value will be added to the Secret's data key, each value should be raw and NOT base64
+  # encoded. Any secrets can be provided here. It's commonly used for credentials and other access related values.
+  # **NOTE: Don't commit unencrypted secrets to git!**
+  generic: {}
+    # my_variable: "my-secret-value"
+    # datadog_api_key: "api-key"
+    # awsAccessKeyId: "access-key"
+    # awsSecretAccessKey: "secret-access-key"
+
+autoscaling:
```

Comment:
autoscaling is not necessary for agent https://github.com/WhizardTelemetry/vector-helm-charts/pull/2/files#diff-033177f7a15cb8a53378f2a323022c089d4e39f4601c9469a6b25457436c4b16R60-R86
```shell
  # autoscaling.enabled -- Create a [HorizontalPodAutoscaler](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/)
  # for Vector. Valid for the "Aggregator" and "Stateless-Aggregator" roles.
````

Reply:
The default configurations copied from values are all false.